src/img/close-gray-90-alt.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What do you think about compressing the SVGs so it matches the compression of the rest of the SVGs? e.g. https://github.com/uswds/uswds/blob/develop/src/img/angle-arrow-down.svg?short_path=87cfc3a

The tool I used was: https://jakearchibald.github.io/svgomg/

Reply:
SVGs don't need more of the extra generated text like "Created with Sketch."

Reply:
yup, will do.